src/utils/localStorage.ts

```diff
@@ -0,0 +1,3 @@
+export const getAccessTokenKey: (venueId: string | undefined) => string = (
+  venueId
+) => `token-${venueId}`;
```

Comment:
If we have a function here for getting an access token key, should we not also have some kind of functions about getting the access token itself? And/or setting it?

Reply:
This is not a getter, it only builds the string to get the token. I just created a getter in `auth.ts`

Reply:
_nods_ I realise this wasn't a getter. Added some new comments in my latest review, shall resolve this.